src-tauri/src/settings.rs

```diff
@@ -104,3 +116,59 @@ pub fn check(required: Vec<Permission>) -> Option<String> {
         return Some(format!("You MUST tell the user they need to allow access to ALL of the following features in settings: {}", all_denied.join(", ")))
     }
 }
+
+pub fn get_settings_file_path() -> PathBuf {
```

Comment:
Are we switching over to calling them settings instead of permissions?

Reply:
It's up for debate of course. I was just thinking that it'd be nice to have a settings module that encompasses everything in the backend for the settings modal, especially as it grows with more things. If its something that you think would be best separated out then we can do that also. currently permissions is a separate json file ("permissions.json") and the struct of course is implemented in the settings.rs. the audio devices stuff is also in settings.rs but there wasn't much to store in a json other than the previously selected audio devices, so I just threw it under a "settings.json". We could combine the permissions.json into the settings.json but the update file function would be a bit more complicated, as currently those are both activated via separate useEffects on the frontend with dependencies of their respective frontend representations. I hope this all makes sense, lmk what you think

Reply:
I am good with renaming them! It was a tad bit confusing with them being together, but I understand your reasoning behind it so I am cool with it